packages/terra-consumer-nav/src/components/nav-help/NavHelp.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Arrange from 'terra-arrange';
+import Grid from 'terra-grid';
+import Popup from 'terra-popup';
+import IconClose from 'terra-icon/lib/icon/IconClose';
+import NavHelpContent from './NavHelpContent';
+import IconInfo from '../../icons/IconInfo';
+import styles from './NavHelp.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  help: PropTypes.arrayOf(PropTypes.shape({
+    slug: PropTypes.string,
+    nav_type: PropTypes.oneOf(['GROUPING', 'EXTERNAL', 'APPLICATION', 'MODAL']),
+    target: PropTypes.string,
+    text: PropTypes.string.isRequired,
+    uri: PropTypes.string.isRequired,
+    icon: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.element,
+    ]),
+    children: PropTypes.arrayOf(PropTypes.shape({
+      slug: PropTypes.string,
+      nav_type: PropTypes.oneOf(['GROUPING', 'EXTERNAL', 'APPLICATION', 'MODAL']),
+      text: PropTypes.string.isRequired,
+      uri: PropTypes.string.isRequired,
+      icon: PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.element,
+      ]),
+      children: PropTypes.array,
+    })),
+  })),
+};
+
+const defaultProps = {
+  help: [],
+};
+
+const translations = {
+  help: 'Help',
+};
+
+class NavHelp extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      isOpen: false,
+    };
+    this.openPopup = this.openPopup.bind(this);
+    this.closePopup = this.closePopup.bind(this);
+  }
+
+  openPopup() {
+    this.setState({ isOpen: true });
+  }
+
+  closePopup() {
+    this.setState({ isOpen: false });
+  }
+
```

Comment:
since the popup is either open or close, we can use one callback handling the state. e.g. https://github.cerner.com/HealtheLife/r_wellness/blob/master/client/examples/components/modal/DefaultModal.jsx#L12